server/src/test/java/org/bonitasoft/web/rest/server/api/bpm/flownode/ActivityVariableResourceTest.java

```diff
@@ -91,36 +91,35 @@ public void should_return() throws DataNotFoundException {
         assertThat(dataInstanceResult).isEqualTo(dataInstance);
     }
 
-    private DataInstanceImpl createShortTextDataInstance(String value) {
-        DataDefinition dataDefinition = createDataDefinition();
+    private DataInstanceImpl createShortTextDataInstance(final String value) {
```

Comment:
Are all those `final` necessary? Thought I heard they were out of the formatting template.